tests/test_EVMAssembler.py

```diff
@@ -44,6 +44,28 @@ def test_JUMPI(self):
         self.assertTrue(insn.mnemonic == 'JUMPI')
         self.assertTrue(insn.is_branch)
 
+    def test_pre_byzantium(self):
+        insn = EVMAsm.disassemble_one(b'\x57', fork='pre-byzantium')
+        self.assertTrue(insn.mnemonic == 'JUMPI')
+        self.assertTrue(insn.is_branch)
+        insn = EVMAsm.disassemble_one(b'\xfd', fork='pre-byzantium')
```

Comment:
Maybe we should iterate over all bytes that should be invalid and assert them.

Reply:
Done. All invalid are tested now.